src/main/webapp/assets/modules/common/utils.fac.js

```diff
@@ -2,6 +2,21 @@ angular.module("Common").factory("utils", function (jsonValue, $location, $rootS
   var techlooperObserver = $.microObserver.get("techlooper");
 
   var instance = {
+    visit: function (obj, func) {
+      for (var prop in obj) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.